jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpCookie.java

```diff
@@ -137,7 +146,14 @@ public HttpCookie(String setCookie)
         _version = cookie.getVersion();
         _expiration = _maxAge < 0 ? -1 : System.nanoTime() + TimeUnit.SECONDS.toNanos(_maxAge);
         // support for SameSite values has not yet been added to java.net.HttpCookie
-        _sameSite = getSameSiteFromComment(cookie.getComment());
+        SameSite sameSite = getSameSiteFromComment(cookie.getComment());
+        if (sameSite != null)
+        {
+            _attributes = new TreeMap<String, String>(String.CASE_INSENSITIVE_ORDER);
+            _attributes.put("SameSite", sameSite.getAttributeValue());
+        }
+        else
+            _attributes = Collections.emptyMap();
```

Comment:
Can we keep the `_sameSite` field.  It will commonly be set and it would be good to avoid creating a map with an attribute for every HttpCookie.  We should only create the map for unknown attributes.   We already skip sameSite attribute when creating the cookie.